examples/cop-dem-glo-30/collection.json

```diff
@@ -0,0 +1,97 @@
+{
+  "type": "Collection",
+  "id": "cop-dem-glo-30",
+  "stac_version": "1.0.0",
+  "description": "The Copernicus DEM is a Digital Surface Model (DSM) which represents the surface of the Earth including buildings, infrastructure and vegetation. We provide two instances of Copernicus DEM named GLO-30 Public and GLO-90. GLO-90 provides worldwide coverage at 90 meters. GLO-30 Public provides limited worldwide coverage at 30 meters because a small subset of tiles covering specific countries are not yet released to the public by the Copernicus Programme. Note that in both cases ocean areas do not have tiles, there one can assume height values equal to zero. Data is provided as Cloud Optimized GeoTIFFs and comes from Copernicus DEM 2021 release.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "./collection.json",
+      "type": "application/json",
+      "title": "Copernicus DEM GLO-30"
+    },
+    {
+      "rel": "handbook",
+      "href": "https://object.cloud.sdsc.edu/v1/AUTH_opentopography/www/metadata/Copernicus_metadata.pdf",
+      "type": "application/pdf",
+      "title": "Copernicus DEM User handbook",
+      "description": "Also includes data usage information"
+    },
+    {
+      "rel": "license",
+      "href": "https://spacedata.copernicus.eu/documents/20126/0/CSCDA_ESA_Mission-specific+Annex.pdf",
+      "type": "Copernicus Data Access"
+    },
+    {
+      "rel": "item",
+      "href": "./Copernicus_DSM_COG_10_N53_00_W115_00_DEM/Copernicus_DSM_COG_10_N53_00_W115_00_DEM.json",
+      "type": "application/json"
+    }
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/projection/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/raster/v1.1.0/schema.json"
```

Comment:
I still don't see any `raster:` attributes in this collection, so we should either add some (e.g. `raster:bands` on the `item_assets`) or remove this schema.

Reply:
odd, I added https://github.com/developmentseed/cop-dem/blob/fee76e0797db5ef4a2d2efaa3c350600b23289a6/src/stactools/cop_dem/stac.py#L84-L89 I'll go figure out why that didn't make it into the examples at the end.

Reply:
fixed, I think there was a silent failure on permissions when I ran the new create_example.py that failed to overwrite the old version.